runtime_test.go

```diff
@@ -665,6 +665,49 @@ func TestRuntime_Close_ClosesCompiledModules(t *testing.T) {
 	}
 }
 
+func TestRuntime_Close_Instantiate_ShouldErr(t *testing.T) {
+	for _, tc := range []struct {
+		name            string
+		instantiateFunc func(r Runtime, mod CompiledModule) error
```

Comment:
`instantiateFunc` is not representative anymore (CompileModule is nothing to do with instantiation)